addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1620,6 +1620,53 @@ QUnit.test('chat window with a thread: keep scroll position in message list on f
     );
 });
 
+QUnit.test('chat window should scroll at the end after a new message is posted', async function (assert) {
+    assert.expect(1);
+
+    this.data['mail.channel'].records.push({
+        id: 20,
+        is_minimized: true,
+        state: 'open',
+        is_pinned: true,
+    });
+    for (let i = 0; i < 10; i++) {
+        this.data['mail.message'].records.push({
+            channel_ids: [20],
+            model: 'mail.channel',
+            res_id: 20,
+        });
+    }
+    await this.start({
+        env: {
+            browser: {
+                innerWidth: 1920,
+            },
+        },
+    });
+
+    // Set content of the composer of the chat window
+    await afterNextRender(() => {
+        document.querySelector(`.o_ComposerTextInput_textarea`).focus();
```

Comment:
Does the test work without the browser being focused? Probably yes, but to double check (and add focus tag in test name if necessary)

Reply:
It does work without focus

Reply:
perfect